pkg/ccl/logictestccl/testdata/logic_test/partitioning_constrained_scans

```diff
@@ -317,3 +317,4 @@ select
  │         └── [/'foo'/e'bar\x00'/5 - ]
  └── filters
       └── c = 5
+
```

Comment:
This was later removed in release-22.2 by https://github.com/cockroachdb/cockroach/commit/bd52adcc56271cbe63d7113a509134f030236c42

Compare via:

- https://github.com/cockroachdb/cockroach/commits/release-22.2.0/pkg/ccl/logictestccl/testdata/logic_test/partitioning_constrained_scans
- https://github.com/cockroachdb/cockroach/commits/release-22.2/pkg/ccl/logictestccl/testdata/logic_test/partitioning_constrained_scans